packages/core/src/Stores/traders-hub-store.js

```diff
@@ -18,6 +18,7 @@ export default class TradersHubStore extends BaseStore {
     selected_account_type;
     selected_region;
     is_onboarding_visited = false;
+    is_first_time_visit = true;
```

Comment:
why don't we just use react state for this? since we're trying to move everything out of mobx store.

Reply:
@shayan-deriv since the current logic is also in the mobx store, I don't think separating out this 1 state variable out of the store could make much difference here. Once we decide to remove this store, we could easily refactor this 1 value in the store

Reply:
Ok . thanks